client/src/components/payment/refund-summary.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <v-expand-transition>
+    <!-- do not display until payments are fetched -->
+    <section id="refund-summary" v-if="payments.length > 0">
+
+      <header>Name Request Refund</header>
+
+      <!-- iterate over line items -->
+      <!-- NB: each line item can contain several fees -->
+      <template v-for="item in lineItems">
+        <ul class="fee-list" :key="item.id">
+          <li class="fee-list__item text-body-1" v-if="item.filingFees">
+            <div class="fee-list__item-name">{{mapItemName(item)}}</div>
+            <div class="fee-list__item-value">${{item.filingFees.toFixed(2)}}</div>
+          </li>
+          <li class="fee-list__item text-body-1" v-if="item.priorityFees">
+            <div class="fee-list__item-name">Priority fee</div>
+            <div class="fee-list__item-value">${{item.priorityFees.toFixed(2)}}</div>
+          </li>
+          <li class="fee-list__item text-body-1" v-if="item.serviceFees">
+            <div class="fee-list__item-name">Service fee</div>
+            <div class="fee-list__item-value">${{item.serviceFees.toFixed(2)}}</div>
+          </li>
+        </ul>
+      </template>
+
+      <footer class="d-flex justify-space-between">
+        <div>Total Refund Amount (CAD)</div>
+        <div>${{total.toFixed(2)}}</div>
+      </footer>
+
+    </section>
+  </v-expand-transition>
+</template>
+
+<script lang="ts">
+import { Component, Prop, Vue } from 'vue-property-decorator'
+
+const _flatten = require('lodash/flatten')
+
+@Component({})
+export default class RefundSummary extends Vue {
+  @Prop({ default: () => [] })
+  readonly payments: any[]
+
+  /** The list of line items in all completed payments. */
+  private get lineItems (): any[] {
+    const lineItems = _flatten(
+      this.payments.map(p => (p.sbcPayment.statusCode === 'COMPLETED') ? p.sbcPayment.lineItems : [])
+    )
+    return lineItems
+  }
+
+  /** The total of all the fees. */
+  private get total (): number {
+    const fees = _flatten(
+      // for now we only care about Filing, Priority and Service fees
+      this.lineItems.map(li => [li.filingFees, li.priorityFees, li.serviceFees])
+    )
+    const total = fees.reduce((t, n) => (t + n), 0)
+    return total
```

Comment:
This returns 0 if payments is empty ([ ]).

Reply:
Why use lodash for a single function instead of map + reduce, just another dep to keep up to date?

Reply:
That's a great question and one that we thought about in the other Entities projects.

As you pointed out, Lodash is already used by the UI. That can be worked around by importing just the Lodash functions used (ie, https://www.npmjs.com/package/lodash.debounce and https://www.npmjs.com/package/lodash.flatten).

But then, if you look deeper, you'll find that a lot of our dependencies use Lodash anyway (eg, Babel, TS/eslint, webpack, sbc-common-component, etc, etc).

However, I agree with you so will look at using map() instead of flatten() !

Reply:
Updated in latest commit!